platforms/cpuinfo.go

```diff
@@ -74,27 +74,40 @@ func getCPUInfo(pattern string) (info string, err error) {
 }
 
 func getCPUVariant() string {
-	variant, err := getCPUInfo("Cpu architecture")
-	if err != nil {
-		log.L.WithError(err).Error("failure getting variant")
-		return ""
-	}
+	var variant string
+	if runtime.GOOS == "windows" {
```

Comment:
This looks good, but given the assumptions that led to it are just in this PR's comments, it might be good to summarize why these 'shortcuts' can be made on Windows (that only v7 is supported on arch == arm) in a brief comment at the top of the switch.

Reply:
Amended